pkg/image/registry/imagestreamimage/rest.go

```diff
@@ -54,7 +46,7 @@ func ParseNameAndID(input string) (name string, id string, err error) {
 // Get retrieves an image by ID that has previously been tagged into an image stream.
 // `id` is of the form <repo name>@<image id>.
 func (r *REST) Get(ctx kapi.Context, id string) (runtime.Object, error) {
-	name, imageID, err := ParseNameAndID(id)
+	name, imageID, err := parseNameAndID(id)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Good catch.
On 05/26/2016 10:34 AM, Maciej Szulik wrote:
>
> In pkg/image/registry/imagestreamimage/rest.go <https://github.com/openshift/origin/pull/8195#discussion_r64709018>:
>
> > @@ -54,7 +46,7 @@ func ParseNameAndID(input string) (name string, id string, err error) {
> >  // Get retrieves an image by ID that has previously been tagged into an image stream.
> >  // `id` is of the form <repo name>@<image id>.
> >  func (r *REST) Get(ctx kapi.Context, id string) (runtime.Object, error) {
> > -	name, imageID, err := ParseNameAndID(id)
> > +	name, imageID, err := parseNameAndID(id)
> >  	if err != nil {
> >  		return nil, err
> >  	}
>
> There's one more candidate for |api.JoinImageStreamTag(name, tag)| in line 88 of this file, where ISI is constructed.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/openshift/origin/pull/8195/files/fe16eb1279f6614acaba33b1106c090b6db87e5f..1a47571e9af44032c6cf88b37e5a92be1a89a6f3#r64709018>
>